widgets/deployments/src/widget.tsx

```diff
@@ -143,14 +183,16 @@ Stage.defineWidget({
         });
     },
 
-    render(widget, data, error, toolbox) {
+    render(widget, data, _error, toolbox) {
         const { Loading } = Stage.Basic;
         const {
             configuration: { showFirstUserJourneyButtons }
         } = widget;
-        if (isEmpty(data)) {
+        if (isEmpty(data) || !data) {
```

Comment:
Maybe we could refactor so that only `!data` check is required here?

Reply:
I suggest to use the following solution:
```suggestion
        if (Stage.Utils.isEmptyWidgetData(data)) {
```
You can find it in other widgets as well.